sandbox/src/mocks/stu3/changeShortlist/requests/NonMixedShortlistWithDocument.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
I don't see these new examples specified in the story. Or do they come from a different story?

Reply:
I added because the existing examples didn't have a DocumentReference